src/games/strategy/triplea/ai/proAI/ProPurchaseAI.java

```diff
@@ -1751,7 +1743,7 @@ private void purchaseSeaAndAmphibUnits(final Map<Territory, ProPurchaseTerritory
         }
 
         // Check to see if its worth trying to defend the territory
-        if (result.getTUVSwing() < 0 || result.getWinPercentage() < WIN_PERCENTAGE) {
+        if (result.getTUVSwing() < 0 || result.getWinPercentage() < ProData.winPercentage) {
```

Comment:
This logic could be moved to ProData nicely, `public static boolean worthTryingToDefendTerritory(Result)`